privaterelay/settings.py

```diff
@@ -915,13 +915,38 @@ def set_index_cache_control_headers(
 if SENTRY_ENVIRONMENT == "prod" and SITE_ORIGIN != "https://relay.firefox.com":
     SENTRY_ENVIRONMENT = "local"
 
+SENTRY_PROD_TRACE_SAMPLE_RATE = config(
+    "SENTRY_PROD_TRACE_SAMPLE_RATE", 0.001, cast=float
+)
+
+
+def sentry_traces_sampler(_) -> float:
+    if SENTRY_ENVIRONMENT == "prod":
+        return SENTRY_PROD_TRACE_SAMPLE_RATE
+    return 1.0
+
+
+# The profiles_sample_rate setting is *relative to* the traces_sample_rate setting. So,
+# this value is multiplied by the traces sample rate. So the profile rates are:
+# local: 1*1 = 100%
+# dev: 1*1 = 100%
+# stage: 0.1*1 = 10% of requests are profiled
+# prod: 0.1*0.001 = 1% of requests are profiled
+def sentry_profile_sampler(_) -> float:
+    if SENTRY_ENVIRONMENT in ["local", "dev"]:
```

Comment:
```suggestion
    if SENTRY_ENVIRONMENT in ["local", "development"]:
```
When I go to Sentry and look for all the environments under the Relay project, instead of `dev` I see `development`.